RequestManagementSystem/private/RequestValidator.py

```diff
@@ -77,45 +78,50 @@ class RequestValidator( object ):
   __metaclass__ = DIRACSingleton
 
   # # dict with required attrs
-  reqAttrs = { "ForwardDISET" : { "Operation": [ "Arguments" ], "Files" : [] },
-               "PutAndRegister" : { "Operation" : [ "TargetSE" ], "Files" : [ "LFN", "PFN" ] },
-               "ReplicateAndRegister" : { "Operation" : [ "TargetSE" ], "Files" : [ "LFN" ] },
-               "PhysicalRemoval" : { "Operation" : ["TargetSE" ], "Files" : [ "PFN" ] },
-               "RemoveFile" : { "Operation" : [], "Files" : [ "LFN" ] },
-               "RemoveReplica" : { "Operation" : [ "TargetSE" ], "Files" : [ "LFN" ] },
-               "ReTransfer" : { "Operation" : [ "TargetSE" ], "Files" : [ "LFN", "PFN" ] },
-               "RegisterFile" : { "Operation" : [ ], "Files" : [ "LFN", "PFN", "ChecksumType", "Checksum", "GUID" ] },
-               "RegisterReplica" : { "Operation" : [ "TargetSE" ], "Files" : [ "LFN", "PFN" ] } }
+  reqAttrs = {
+      "ForwardDISET": {
+          "Operation": ["Arguments"], "Files": []}, "PutAndRegister": {
```

Comment:
These new line breaks are terrible.
Also if I add new Request Types in my extension, do I need to extend the RequestValidator?